src/components/shared/translations.js

```diff
@@ -86,7 +86,8 @@ const de = {
     add: 'Hinzufügen',
     delete: 'Löschen',
     deselect: {
-      all: 'Alle abwählen'
+      all: 'Alle abwählen',
+      element: 'Auswahl aufheben'
```

Comment:
The terms are basically expressing the same thing. `operations.deselect.all` is used in DpMultiSelect and DpTreeList for the same thing. I would prefer the term "Auswahl aufheben" over "Alle abwählen", but i'd use the key `operations.deselect.all` and drop the other key.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/e768e6fbc47ad63a2e2be07bbbc708b9be23bb45